workflow-templates/automerge.yml

```diff
@@ -0,0 +1,23 @@
+# Automatically merge pull requests opened by ipldbot, as soon as (and only if) all tests pass.
+# This reduces the friction associated with updating with our workflows.
+
+on: [ pull_request ]
+
+jobs:
+  automerge:
+    if: github.event.pull_request.user.login == 'ipldbot'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Wait on tests
+      uses: lewagon/wait-on-check-action@v0.2
```

Comment:
We need to fork these actions into our org. Using externally defined actions is insecure.

Reply:
Insecure == the author can re-tag and then get a "token" that will let them do pretty much anything.